src/nomad_simulations/properties/energies.py

```diff
@@ -77,3 +78,502 @@ def __init__(
+    calculated using the functional stored in XC_functional.
+    """
+    # ! Someone check this description!
+    # ? Do we really want to specify the method here? This can't be user-defined?
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+
+class XCPotentialEnergy(QuantumEnergy):
+    """
+    Physical property section describing the potential energy contribution to the exchange-correlation (XC) energy,
+    i.e., the integral of the first order derivative of the functional
+        stored in XC_functional (integral of v_xc*electron_density), i.e., the component
+        of XC that is in the sum of the eigenvalues. Value associated with the
+        configuration, should be the most converged value.          |
+    """
+    # ! Someone check this description!
+    # ? Do we really want to specify the method here? This can't be user-defined?
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+
+# ? XCCorrelationEnergy?
+class CorrelationEnergy(QuantumEnergy):
+    """
+    Physical property section describing the correlation energy of a (sub)system,
+    calculated using the method described in XC_functional.
+    """
+
+    # ! Someone check this description!
+    # ? Do we really want to specify the method here? This can't be user-defined?
+
+    value = Quantity(
+        type=np.float64,
+        unit='joule',
+        description="""
+        The value of the correlation energy.
+        """,
+    )
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+
+# ? XCExchangeEnergy?
+class ExchangeEnergy(QuantumEnergy):
+    """
+    Physical property section describing the exchange energy of a (sub)system,
+    calculated using the method described in XC_functional.
+    """
+    # ! Someone check this description!
+    # ? Do we really want to specify the method here? This can't be user-defined?
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+
+class ZeroTemperatureEnergy(QuantumEnergy):
+    """
+    Physical property section describing the total energy of a (sub)system extrapolated to $T=0$, based on a free-electron gas argument.
+    """
+    # ! Someone check this description!
+    # ? Do we really want to specify the method here? This can't be user-defined?
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+
+class ZeroPointEnergy(QuantumEnergy):
+    """
+    Physical property section describing the zero-point vibrational energy of a (sub)system,
+    calculated using the method described in zero_point_method.
+    """
+    # ! Someone check this description!
+    # ? Do we really want to specify the method here? This can't be user-defined?
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
```

Comment:
Quantum-specific subtype of vibrational energy.

Reply:
Can you be more specific what you suggest changing in this case? I need your guidance for the quantum contributions (maybe it needs to wait until we figure out whether or not to distinguish Quant/Class though)

Reply:
The name is fine, I'd alter the subtype to `ZeroPointEnergy(VibrationalEnergy)`.
This is the lowest vibrational state, which has a non-zero contribution.